src/lib/filehelpers.ts

```diff
@@ -0,0 +1,19 @@
+export function appropriateFileSize(fileSize: number): string {
+	const c = 1024
+	const dp = 2 //number of decimal places
+	const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```

Comment:
Support for Yobibyte already - nice... 😱

Reply:
just in case 😃 #scalable